integration_tests/test_cycles_eth.py

```diff
@@ -26,23 +26,27 @@
 
 
 @pytest.fixture(autouse=True)
-def mock_fns(monkeypatch):
-    monkeypatch.setattr("rewards.calc_rewards.download_boosts", mock_download_boosts)
-    monkeypatch.setattr(
+def mock_fns(mocker):
+    mocker.patch("rewards.calc_rewards.download_boosts", mock_download_boosts)
+    mocker.patch(
         "rewards.calc_rewards.download_proposed_boosts", mock_download_boosts
     )
-    monkeypatch.setattr("rewards.calc_rewards.upload_boosts", mock_upload_boosts)
-    monkeypatch.setattr(
+    mocker.patch("rewards.calc_rewards.upload_boosts", mock_upload_boosts)
+    mocker.patch(
         "rewards.calc_rewards.upload_proposed_boosts", mock_upload_boosts
     )
-    monkeypatch.setattr("rewards.calc_rewards.upload_tree", mock_upload_tree)
+    mocker.patch("rewards.calc_rewards.upload_tree", mock_upload_tree)
+    mocker.patch(
+        "rewards.snapshot.claims_snapshot.get_claimable_data", mock_get_claimable_data
+    )
 
 
 @pytest.fixture(autouse=True)
-def set_env_config(monkeypatch):
+def set_env_config(mocker):
     env_config.test = False
     env_config.staging = False
     env_config.production = True
+    env_config.get_web3 = lambda chain: web3
```

Comment:
What's the purpose of `lambda chain: web3`?

Reply:
Yes, but why you need to pass `chain` as param to lambda?

Reply:
because the get_web3 function has that as the param